scripts/python/proposers_rewards.py

```diff
@@ -514,7 +531,12 @@ def calculate_rewards(
         )
 
         challenge_output_file_path = build_path_for_challenge(
-            output_file, challenge.title.replace(" ", "_").replace(":", "_")
+            output_file,
+            re.sub(
+                r'(?u)[^-\w.]',
```

Comment:
what is this regex supposed to do?

Reply:
it sanitizes the file name that will be used, the filenames generated are based on the name of the challenges. We already encountered problems in win environments.